src/gui/owncloudgui.cpp

```diff
@@ -89,6 +89,16 @@ ownCloudGui::ownCloudGui(Application *parent)
 
     connect(_tray.data(), &Systray::shutdown,
         this, &ownCloudGui::slotShutdown);
+    connect(_tray.data(), &Systray::openShareDialog,
+        this, [=](const QString &sharePath, const QString &localPath, const bool publicLink = false)
+            {
+
+                if (publicLink) {
+                    this->slotShowShareDialog(sharePath,localPath, ShareDialogStartPage::PublicLinks);
```

Comment:
using this-> looks weird to me, but I'm coming from the outside so ignore me if it is consistent with the other code :)

Reply:
Agreed, "this->" is foreign. Note that the desktop client code base is not consistent in that regard, it uses "this->" a couple of times, but not most of the time... So let's reduce the "this->" use not increase it so that it'll be consistent eventually. :-)

Reply:
This sounds as if it was just declared as the new standard for this project, so I won't argue with that - I'll remove this this (...) and all the other ones that exist within my PRs.

Reply:
Well, as I mentioned we're not currently 100% consistent indeed. But "this->" being the clear minority after grepping, I'm siding on dropping it. ;-)